Doc/library/os.rst

```diff
@@ -4167,45 +4167,55 @@ used to determine the disposition of a process.
    Return ``True`` if a core dump was generated for the process, otherwise
    return ``False``.
 
+   This function should be employed only if :func:`WIFSIGNALED` returned true.
```

Comment:
```suggestion
   This function should be employed only if :func:`WIFSIGNALED` returned ``True``.
```

Reply:
Same for others what do you think?

Reply:
Hum, I copied the code from the manual page which is written for C code. I rephased it to:

```
only if :func:`WIFSIGNALED` is true
```